config/katello.yaml

```diff
@@ -9,6 +9,9 @@
   :installer_dir: "."
   :default_values_dir: "/tmp"
   :module_dirs: ["./_build/modules", "../katello-installer/modules"]
+  :parser_cache_path:
+    - /usr/share/foreman-installer/parser_cache/foreman.yaml
+    - ./_build/parser_cache/katello.yaml
```

Comment:
I think if we set this to the following we avoid having to change this during RPM build time:

```
../katello-installer/parser_cache/katello.yaml
```